contrib/docker-device-tool/README.md

```diff
@@ -0,0 +1,11 @@
+Docker device tool for devicemapper storage driver backend
+===================
+
+The ./contrib/docker-device-tool contains a tool to manipulate devicemapper thin-pool.
+
+Compile
+========
+
+    $ make shell
+    ## inside build container
+    $ go build -tags libdm_no_deferred_remove contrib/docker-device-tool/device_tool.go
```

Comment:
That's specific to certain images. I could build device_tool on fedora host and there it should build just fine without this tag. You might want to specifically mention that if devicemapper version is old and compliation fails, use this tag.